tools/licenses/test/formatter_test.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:licenses/formatter.dart';
+import 'package:litetest/litetest.dart';
+
+void main() {
+  test('Block comments', () {
+    expect(reformat('/* test */'), 'test');
+    expect(reformat('/*\ntest\n*/'), 'test');
+    expect(reformat('/*\ntest\n */'), 'test');
+    expect(reformat('/*\n * test\n*/'), 'test');
+    expect(reformat('/*\n * test\n */'), 'test');
+    expect(reformat('/*\n * test\n * test\n */'), 'test\ntest');
+    expect(reformat('/*\n * test\n + test\n */'), '* test\n+ test');
+    expect(reformat('test */'), 'test');
+  });
+  test('Indenting blocks', () {
+    expect(reformat('  a\nb\n  c'), 'a\nb\n  c');
+    expect(reformat('  a\n b\n  c'), 'a\nb\nc');
```

Comment:
Some of these test lines could use comments; I don't know why Adding one space before `b` is expected to remove two spaces before `c` for instance (same for similar jumps in other tests).

Reply:
added comments to all the lines in "Indenting blocks" section.